File: src/NEFBDAAStarter/angular/projects/nefbdaa-angular-nebular/src/lib/models/api/exception-ui-config.ts
export interface ExceptionUiConfig {
  timeout?: number;
  type?: ExceptionUiTypeModel;
}


type ExceptionUiTypeModel =
  'NOTIFICATION' |
  'DIALOG';
module ExceptionUiType {
  export const NOTIFICATION: ExceptionUiTypeModel = 'NOTIFICATION';
  export const DIALOG: ExceptionUiTypeModel = 'DIALOG';

  export function values(): ExceptionUiTypeModel[] {
    return [
      NOTIFICATION,
      DIALOG,
    ];
  }
}

export {ExceptionUiTypeModel}

File: NEFBDAACommons/Shared/Exceptions/ExceptionUiConfig.cs
namespace NEFBDAACommons.Shared.Exceptions
{
    public enum ExceptionUiType
    {
        NOTIFICATION, DIALOG
    }
    public class ExceptionUiConfig
    {
        public int Timeout { get; set; } = 5000;
        public ExceptionUiType Type { get; set; } = ExceptionUiType.NOTIFICATION;
    }
}
